packages/loader/container-loader/src/debugLogger.ts

```diff
@@ -81,9 +81,9 @@ export class DebugLogger implements ITelemetryBaseLogger {
 
 		// Use debug's coloring schema for base of the event
 		const index = event.eventName.lastIndexOf(eventNamespaceSeparator);
-		const name = event.eventName.substring(index + 1);
+		const name = event.eventName.slice(Math.max(0, index + 1));
 		if (index > 0) {
-			logger = logger.extend(event.eventName.substring(0, index));
+			logger = logger.extend(event.eventName.slice(0, Math.max(0, index)));
```

Comment:
Inside `if (index > 0)` the Math.max bit shouldn't be necessary, right? And two lines above we're pretty sure it's also non-negative, so I don't think it's necessary either.